flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/factories/TableFactoryUtil.java

```diff
@@ -196,4 +204,92 @@ public ClassLoader getUserClassLoader() {
                                                 }))
                 .collect(Collectors.toList());
     }
+
+    /**
+     * Finds and creates a {@link CatalogStore} using the provided {@link Configuration} and user
+     * classloader.
+     *
+     * <p>The configuration format should be as follows:
+     *
+     * <pre>{@code
+     * table.catalog-store.kind: {identifier}
+     * table.catalog-store.{identifier}.{param1}: xxx
+     * table.catalog-store.{identifier}.{param2}: xxx
+     * }</pre>
+     */
+    public static CatalogStore findAndCreateCatalogStore(
```

Comment:
I think this part should be put into `FactoryUtil` in `flink-table-common`.

Reply:
The flink-table-api-scala-bridge module does not have a dependency on flink-table-common, so I added it here.